lib/pages/wp-admin/wp-admin-jetpack-page.js

```diff
@@ -13,8 +16,23 @@ export default class WPAdminJetpackPage extends AsyncBaseContainer {
 
 	async connectWordPressCom() {
 		const selector = By.css( 'a.jp-jetpack-connect__button' );
+		const pauseBetweenFocusAttemptsMS = 200;
+		const connectJetpackButtonFocus =
+			"document.getElementsByClassName( 'jp-jetpack-connect__button')[0].focus();";
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
-		await this.driver.sleep( 1000 );
+		for ( let i = 0; i < explicitWaitMS / pauseBetweenFocusAttemptsMS; i++ ) {
+			await this.driver.executeScript( connectJetpackButtonFocus );
+			let currentActiveElementId = await this.driver
+				.switchTo()
+				.activeElement()
+				.getId();
+			let connectJetpackButtonId = await ( await this.driver.findElement( selector ) ).getId();
```

Comment:
are these params around `findElement` is needed? I think chained call with single `await` should do the job

Reply:
You are right, single `await` would do the job here. Thanks for catching it!